examples/project_from_cloud_volume_mesh.py

```diff
@@ -0,0 +1,27 @@
+import flow360.component.simulation.units as u
+from flow360.component.project import Project
+from flow360.component.simulation.models.surface_models import Freestream, Wall
+from flow360.component.simulation.models.volume_models import Fluid
+from flow360.component.simulation.operating_condition.operating_condition import (
+    AerospaceCondition,
+)
+from flow360.component.simulation.simulation_params import SimulationParams
+from flow360.component.simulation.unit_system import SI_unit_system
+from flow360.environment import dev
+
+dev.active()
+
+project = Project.from_cloud("prj-e8c6c7eb-c18b-4c15-bac8-edf5aaf9b155")
+
+volume_mesh = project.volume_mesh
+
+with SI_unit_system:
+    params = SimulationParams(
+        operating_condition=AerospaceCondition(velocity_magnitude=100 * u.m / u.s),
+        models=[
+            Wall(entities=[volume_mesh["fluid/wall"]]),
+            Freestream(entities=[volume_mesh["fluid/farfield"]]),
+        ],
+    )
+
+project.run_case(params=params)
```

Comment:
let's make it a bit more interesting:
```python
from pylab import show

residuals = project.case.results.nonlinear_residuals
residuals.as_dataframe().plot(x="pseudo_step", logy=True)

show()
```

Reply:
I added it, but it looks like currently there are some issues with dev - no new project goes through and meshMetaData pipeline fails after upload without logs... Will try again in a couple hours.

Since pylab is not part of our dependencies I also changed it to matplotlib.pyplot - from pandas docs it looks like dataframe.plot() uses matplotlib so it should work...

Reply:
Available in [d07cfe8](https://github.com/flexcompute/Flow360/pull/511/commits/d07cfe8777c1386e17c47bd53f7199ae10ce6d0e)
Merge state
This branch has not been deployed
No deployments
Changes approved
1 approving review by reviewers with write access. 
1 approval
This branch has no conflicts with the base branch
Merging can be performed automatically.
 You can also open this in GitHub Desktop or view .
@andrzej-krupka


Add a comment
Comment
 
Add your comment here...
 
Remember, contributions to this repository should follow our GitHub Community Guidelines.
 ProTip! Add comments to specific lines under Files changed.
Reviewers
@maciej-flexcompute
[maciej-flexcompute](https://github.com/maciej-flexcompute)

@[benflexcompute](https://github.com/benflexcompute)
benflexcompute

Still in progress?
Assignees
No one—
Labels
None yet
Projects
None yet
Milestone
No milestone
Development
Successfully merging this pull request may close these issues.

None yet


Notifications
Customize
You’re receiving notifications because you authored the thread.
3 participants
@andrzej-krupka
@maciej-flexcompute
@benflexcompute